### PR TITLE
Release preparation for 2.5.0

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
Update changelog and podspec

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +15,6 @@
 
 - **Feature:** Added implementation for orientation lock
 - **Feature:** Added implementation to retrieve Access token details
----
```
Comment (ts-leojoseph-r) [inline CHANGELOG.md:16]: We don't need separators between each releases?

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (corycaywood) [reply CHANGELOG.md:16]: None of the other releases had separator lines, but I guess we could add this to all of them. I'll update this.

**Review by Climbatize — APPROVED** [review]
👍

**Pull request merged**